edx_when/models.py

```diff
@@ -96,18 +106,23 @@ class ContentDate(models.Model):
     location = UsageKeyField(null=True, default=None, db_index=True, max_length=255)
     field = models.CharField(max_length=255, default='')
     active = models.BooleanField(default=True)
+    block_type = models.CharField(max_length=255, null=True)
+    course_version = models.ForeignKey(CourseVersion, null=True, on_delete=models.CASCADE)
```

Comment:
@ormsbee In thinking more about this field, does it need to be present at all? The code currently does *not* use it to filter against. It's stored for `ContentDate` rows but never used to filter against in the current code - so it's really there just as a piece of info that might be useful for future debugging. The course_version/published_version is currently only used to invalidate old cache keys upon a new course publish.

Reply:
It sounds fine to remove.